pkg/http/api.go

```diff
@@ -2022,6 +2215,21 @@ func GetStatusCodeFromMetadata(metadata map[string]string) int {
 	return fasthttp.StatusOK
 }
 
+func populateMetadataForBulkPublishEntry(reqMeta, entryMeta map[string]string) map[string]string {
+	resMeta := map[string]string{}
```

Comment:
This function is exactly the same as in the `grpc` package. Can you move this to a common package, like utils?

Reply:
Addressed in https://github.com/dapr/dapr/pull/5394